readme.md

```diff
@@ -1,7 +1,7 @@
 # Search bar for Airbnb
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://g37ua.github.io/layout_search-bar-airbnb/)
```

Comment:
![image](https://user-images.githubusercontent.com/38065883/170543817-4dec95f8-9923-40b4-9537-5199e086a747.png)
Placeholder and an image are not aligned vertically